shared/middleware/helpers/asset_bucket.rb

```diff
@@ -1,7 +1,13 @@
+require 'mini_magick'
+
 #
 # AssetBucket
 #
 class AssetBucket < BucketHelper
+  def max_resize_size
```

Comment:
nit: can you add a comment here that this is the size to resize from, not the size to resize to?